libs/platform/src/lib/list/list.component.ts

```diff
@@ -814,30 +822,28 @@ export class ListComponent<T> extends CollectionBaseInput implements OnInit, Aft
 
 @Component({
     selector: 'fdp-list-footer',
-    template: ` <li #listfooter class="fd-list__footer" [attr.id]="id" role="option">
+    template: ` <li #listFooter class="fd-list__footer" [attr.id]="id" role="option">
         <ng-content></ng-content>
     </li>`
 })
-// eslint-disable-next-line @angular-eslint/component-class-suffix
-export class ListFooter extends BaseComponent {}
+export class ListFooterComponent extends BaseComponent {}
 
 @Component({
     selector: 'fdp-list-group-header',
     template: ` <li #listItem fd-list-group-header [attr.id]="id" role="option" [tabindex]="0">
-        <span fd-list-title>{{ grpheaderTitle }}</span>
+        <span fd-list-title>{{ groupHeaderTitle }}</span>
         <ng-content></ng-content>
     </li>`,
-    providers: [{ provide: BaseListItem, useExisting: forwardRef(() => ListGroupHeader) }]
+    providers: [{ provide: BaseListItem, useExisting: forwardRef(() => ListGroupHeaderComponent) }]
 })
-// eslint-disable-next-line @angular-eslint/component-class-suffix
-export class ListGroupHeader extends BaseListItem implements OnInit {
+export class ListGroupHeaderComponent extends BaseListItem implements OnInit {
     /** Displays list goup header title */
     @Input()
-    grpheaderTitle?: string;
```

Comment:
Considering this is a breaking change, maybe leave the old input and have it work as a getter for the new value?